blog/_posts/2016-10-19-scalajs-bundler.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+post-type: blog
+by: Julien Richard-Foy
+title: "scalajs-bundler has been released"
+---
+
+This week we have released a first version of [`scalajs-bundler`](https://scalacenter.github.io/scalajs-bundler), a tool aiming to provide an integrated developer experience for
+[Scala.js](https://www.scala-js.org/) projects that use [npm](https://www.npmjs.com/) packages.
+
+## Motivation
+
+npm is the most popular JavaScript package registry. How can Scala.js applications benefit from
+the many libraries published on npm? First, this requires to resolve these libraries, as well as
+their transitive dependencies, from the npm registry and to download them. Then, the artifacts
+need to be somehow “linked” with the Scala.js code. This latter step is more subtle than it seems
+because the linking process might vary according to the target execution environment. For instance,
+Node.js expects artifacts to be conform to the CommonJS format, whereas this format is not compatible
+with the execution from a Web browser. Furthermore, in the case of Web development, when the
+application is shipped to production it is better to pack all the code and its dependencies into a
+single bundle whose format is executable by Web browsers.
+
+## Existing solutions
+
+There are mainly two existing approaches to tackle these challenges: using
+[WebJars](http://www.webjars.org/) or combining two build systems (one for the Scala
+world and one for the npm world). Both of them require extra efforts from developers
+or have limitations (you can find more details about that in scalajs-bundler’s
+[documentation](https://scalacenter.github.io/scalajs-bundler/motivation.html)).
+
+## Vision, current state and future work
+
+The last release of Scala.js brought support for
+[CommonJS modules](https://www.scala-js.org/doc/project/module.html), thus opening new
+possibilities.
+
+scalajs-bundler aims to make npm dependencies management as easy as JVM dependencies
+management.
+
+Basically, the idea is that if an application needs to depend on an
```

Comment:
I would merge this paragraph with the previous one and reword it as:
"In a nutshell, if you need to depend on a npm package (`foo`), add the following line to your sbt build:"